<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="M2B-D-6ZLLXH2" failures="0" tests="8" name="pracModules.AppEntryGIFT" time="16.851" errors="1" timestamp="14 May 2018 08:50:13 GMT" skipped="7">
  <testcase classname="pracModules.AppEntryGIFT" name="AccountWelcome" time="16.851">
    <error message="Must provide a location for a move action." type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Must provide a location for a move action.at org.openqa.selenium.interactions.MoveMouseAction.<init>(MoveMouseAction.java:38)at org.openqa.selenium.interactions.Actions.moveToElement(Actions.java:388)at pracModules.AppEntryGIFT.AccountWelcome(AppEntryGIFT.java:236)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.base/java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
    </error>
  </testcase> <!-- AccountWelcome -->
  <testcase classname="pracModules.AppEntryGIFT" name="CLTApplicaionOwnerInformationGIFT" time="0.000">
    <skipped/>
  </testcase> <!-- CLTApplicaionOwnerInformationGIFT -->
  <testcase classname="pracModules.AppEntryGIFT" name="NewBusinessApplicationSummary" time="0.000">
    <skipped/>
  </testcase> <!-- NewBusinessApplicationSummary -->
  <testcase classname="pracModules.AppEntryGIFT" name="ApplicationDefingQuestionnaire" time="0.000">
    <skipped/>
  </testcase> <!-- ApplicationDefingQuestionnaire -->
  <testcase classname="pracModules.AppEntryGIFT" name="ApplicationPrimarybeneficiaryInformation" time="0.000">
    <skipped/>
  </testcase> <!-- ApplicationPrimarybeneficiaryInformation -->
  <testcase classname="pracModules.AppEntryGIFT" name="ApplicationPremiumInformation" time="0.000">
    <skipped/>
  </testcase> <!-- ApplicationPremiumInformation -->
  <testcase classname="pracModules.AppEntryGIFT" name="AuthenticationAndAccessControl">
    <skipped/>
  </testcase> <!-- AuthenticationAndAccessControl -->
  <testcase classname="pracModules.AppEntryGIFT" name="LifeInsuranceAndAnnuityPlatform">
    <skipped/>
  </testcase> <!-- LifeInsuranceAndAnnuityPlatform -->
</testsuite> <!-- pracModules.AppEntryGIFT -->
